pkg/libovsdb/schema.go

```diff
@@ -7,6 +7,8 @@ import (
 	"io/ioutil"
 	"os"
 	"strings"
+
+	"k8s.io/klog"
```

Comment:
remove, library code should not log messages (this is for the caller program to decide)